src/subtitling/Group setups/github.md

```diff
@@ -0,0 +1,145 @@
+# Using GitHub for fansubbing
+
+When working in a group,
+you'll often find multiple people making changes to the same or multiple files.
+The Translator will need a place to upload their translation to,
+the typesetter(s) may need a place where they can upload their typesetting to,
+etc.
+For this kind of operation,
+a Git environment may be preferable.
```

Comment:
"git" is actually lower-cased and a link to it would probably be useful to some, albeit not many.